geode-redis/src/main/java/org/apache/geode/redis/internal/data/RedisString.java

```diff
@@ -649,8 +649,9 @@ protected void applyDelta(DeltaInfo deltaInfo) {
         appendSequence = appendDeltaInfo.getSequence();
       } else if (appendDeltaInfo.getSequence() != appendSequence) {
         // Exceptional case should never happen
-        throw new RuntimeException("Redis APPEND sequence mismatch - delta sequence number: "
-            + appendDeltaInfo.getSequence() + " current sequence number: " + appendSequence);
+        throw new RuntimeException(
+            "Compatibility with Redis APPEND sequence mismatch - delta sequence number: "
+                + appendDeltaInfo.getSequence() + " current sequence number: " + appendSequence);
```

Comment:
(remove "Compatibility with Redis" - so it reads: 
APPEND sequence mismatch - delta sequence number:

Does Redis throw an error like this as well?  We should think about what users will need to do if they come across this message.

Reply:
redis does not appear to throw any such exception, and it seems like we will rarely, if ever, throw it. I have no idea what a user would do about it. @ringles and @jhutchison worked on the delta stuff so they may have a better idea of when this will happen